apps/modernization-ui/src/pages/patient/profile/summary/PatientProfileSummary.tsx

```diff
@@ -86,7 +86,12 @@ export const PatientProfileSummary = ({ patient, summary }: Props) => {
             ) : (
                 <>
                     <div className="border-bottom border-base-lighter patient-summary-title">
-                        <h2>{`${formattedName(summary?.legalName?.last, summary?.legalName?.first)}`}</h2>
+                        <h2>{`${formattedName(
+                            `${summary?.legalName?.first || ''} ${summary?.legalName?.middle || ''}  ${
+                                summary?.legalName?.last || ''
+                            }`,
```

Comment:
This is always showing a `,` even when a Patient doesn't have a suffix.

Reply:
@adamloup-enquizit I just tested this and it is not showing , when a patient doesn't have a suffix. Could you please let me know the steps to reproduce it.
<img width="1081" alt="Screenshot 2023-10-04 at 11 46 06 AM" src="https://github.com/CDCgov/NEDSS-Modernization/assets/118309975/73eea795-8032-4b44-ae21-cf4106a5db1e">

Reply:
you are correct!